### DISCUSSION TITLE
Shipping cost by Postal Code

### DISCUSSION THREAD

**Discussion by blackyzero** [vendurehq/vendure#1621] [General]
Hello,

Does Vendure support shipping cost based on Postal Code ? 
If not, could you consider to add this feature?

Thank you.

**Comment by michaelbromley** [comment]
Hi,
Sure, you can support this by implementing a custom ShippingCalculator: https://www.vendure.io/docs/developer-guide/shipping/#creating-a-custom-calculator

**Comment by blackyzero** [comment]
Great thanks @michaelbromley for pointing it out, but i am not coder, so it would become really big work to me.

Would it possible to add this feature in Admin dashboard, so it could be easily to manage?
Ex: 
- shipping zone 1 with Post code start with ABC, will be applied $20
- shipping zone 2 with Post code start with XYZ, will be applied $40

Thank you.

**Reply by michaelbromley** [reply]
If you are not a developer, I'm afraid Vendure is going to be tough going for you to work with without assistance. It is a _developer framework_, so developers are the target audience of Vendure (or at least decision makers who have access to a developer or team).

If you have the means, hire a developer to assist you with your Vendure project. If this is not realistic, you might want to look elsewhere fore a e-commerce platform which is targeted at a non-technical audience such as Shopify.

**Comment by blackyzero** [comment]
Yeah, i understand. It could be hard, but it's worth to try and learn.
Anyway, thanks for this advice. I am checking various options atm, and actually Vendure impresses me, still discovering it :)
